doc/tools/west/repo-tool.rst

```diff
@@ -327,5 +327,47 @@ These commands perform miscellaneous functions.
   that fetches upstream data is run (this behavior can be
   suppressed for the duration of a single command by passing ``--no-update``).
 
+.. _west-mr-build:
+
+Build system integration
+************************
+
+Some of the projects managed by west require to be integrated with the Zephyr
+build system, which is based around Kconfig and CMake (see
+:ref:`application` for more information about Zephyr's build system).
+
+The way that the build system discovers and includes west-managed projects when
+building an application is based around the ``west list`` command. The main
+CMake script in zephyr (the :file:`CMakeLists.txt` located at the zephyr
```

Comment:
Zephyr should be capitalized when it's a reference to the project or OS (and not a directory name).

Reply:
In this case it is referring to the "zephyr repository root folder", so it is a directory name.